sig-scalability/README.md

```diff
@@ -35,18 +35,28 @@ The Chairs of the SIG run operations and processes governing the SIG.
 
 The following subprojects are owned by sig-scalability:
 - **kubernetes-scalability-definition**
+  - Description: [Described below](#kubernetes-scalability-definition)
   - Owners:
     - https://github.com/kubernetes/community/blob/master/sig-scalability/slos/OWNERS
 - **kubernetes-scalability-governance**
+  - Description: [Described below](#kubernetes-scalability-governance)
   - Owners:
     - https://github.com/kubernetes/community/blob/master/sig-scalability/governance/OWNERS
 - **kubernetes-scalability-test-frameworks**
+  - Description: [Described below](#kubernetes-scalability-test-frameworks)
   - Owners:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cmd/kubemark/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/kubemark/OWNERS
     - https://raw.githubusercontent.com/kubernetes/perf-tests/blob/master/clusterloader2/OWNERS
     - https://raw.githubusercontent.com/kubernetes/perf-tests/blob/master/OWNERS
+- **kubernetes-scalability-and-performance-tests-and-validation**
+  - Description: [Described below](#kubernetes-scalability-and-performance-tests-and-validation)
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/e2e/scalability/OWNERS
+- **kubernetes-scalability-bottlenecks-detection**
+  - Description: [Described below](#kubernetes-scalability-bottlenecks-detection)
+  - Owners:
```

Comment:
This owners is empty? Should we mention `/blogs` subdirectory under this?

Reply:
I created https://github.com/kubernetes/community/pull/3054 to address that.